templates/taxi/index.html

```diff
@@ -0,0 +1,8 @@
+{% extends "base.html" %}
+{% block content %}
+    <ul>
+        <li>Number of cars: {{ num_cars }}</li>
+        <li>Number of drivers: {{ num_drivers }}</li>
+        <li>Number of manufacturers: {{ num_drivers }}</li>
```

Comment:
It seems like there is a typo on this line. You are displaying the number of manufacturers but the variable used is `num_drivers`. You should use a variable that contains the number of manufacturers.